windows.ltsc2022/Dockerfile

```diff
@@ -1,45 +1,45 @@
 # escape=`
 
-FROM mcr.microsoft.com/dotnet/framework/runtime:4.8.1-20230808-windowsservercore-ltsc2022
+FROM mcr.microsoft.com/dotnet/framework/runtime:4.8.1-windowsservercore-ltsc2022
```

Comment:
We pin the windows base image here because we previously ran into an issue where these images used too much space on the Windows Dynamic Workers. Windows applies a patch version on top of the base image so security patches are still available. These aren't particularly frequent for Windows which is good and allows some shared caching for windows images. Can we keep this version pinned? Long term we'll need to move away from this or eventually update it, but for now I'd prefer not to roll the dice with using the additional space on dyanmic workers.

https://hub.docker.com/layers/octopusdeploy/worker-tools/windows.ltsc2022/images/sha256-856130db2e5cfe9e0bf52d94ea210b63bf5a7909cebe4206531793863550bc3f?context=explore

https://hub.docker.com/layers/octopusdeploy/worker-tools/6.0-windows.ltsc2022/images/sha256-6cfc7d1319a9b13853d88ea133032336d8d1cae4101cf0c04089cc497b61d582?context=explore